delivery_package_gs1_barcode_report/readme/USAGE.rst

```diff
@@ -0,0 +1,4 @@
+#. Create a new transfer with a product with a serial number.
+#. Assign packages to the individual products in the transfer
+#. On the transfer, click Print SSCC barcode label.
+
```

Comment:
During testing I noted that in order to be able to use this module, the Odoo setting of Settings > Operations > Packages needs to be enabled. Maybe you can write this in the installation instructions.